DirectProgramming/DPC++FPGA/Tutorials/Tools/experimental/platform_designer/Readme.md

```diff
@@ -0,0 +1,274 @@
+
+![](Readme.md.assets/csr-output-example-simple.svg)
+
+This example is intended for users interested in creating standalone modules that can be included in Intel® Quartus® Prime projects. It serves as a minimal example, and while it targets a very specific board, a user familiar with the Intel® Quartus® Prime suite should be able to easily port this design to other hardware.
+
+### Board-specific Considerations
+
+This design is intended to work with the Intel® Arria® 10 SX SoC Development Kit. The board specific configurations are:
+1. Choose `10AS066N3F40E2SG` device to match the devkit
+2. Choose pin `PIN_AM10` to drive the `i_clk` signal
+3. Choose pin `PIN_AR23` to drive the `fpga_led` signal
+4. Choose pin `PIN_AV21` to drive the `reset_button_n` signal
+5. Use `jtag.sdc` from the Intel® Arria® 10 SoC Golden Hardware Reference Design (GHRD) [source code](https://github.com/altera-opensource/ghrd-socfpga).
+
+## Building the `platform_designer` Tutorial
+
+> **Note**: If you have not already done so, set up your CLI
+> environment by sourcing  the `setvars` script located in
+> the root of your oneAPI installation.
+>
+> Linux*:
+> - For system wide installations: `/opt/intel/oneapi/setvars.sh`
+> - For private installations: `~/intel/oneapi/setvars.sh`
+>
+> Windows*:
+> - `C:\Program Files(x86)\Intel\oneAPI\setvars.bat`
+> - For PowerShell*, use the following command: `cmd.exe "/K" '"C:\Program Files (x86)\Intel\oneAPI\setvars.bat" && powershell'`
+>
+>For more information on environment variables, see **Use the setvars Script** for [Linux or macOS](https://www.intel.com/content/www/us/en/develop/documentation/oneapi-programming-guide/top/oneapi-development-environment-setup/use-the-setvars-script-with-linux-or-macos.html), or [Windows](https://www.intel.com/content/www/us/en/develop/documentation/oneapi-programming-guide/top/oneapi-development-environment-setup/use-the-setvars-script-with-windows.html).
+
+Follow these steps to compile and test the design:
+1. Compile the SYCL code to RTL using a Windows or Linux machine. 
+
+   Linux:
+
+   ```bash
+   $> cd add-oneapi
+   $> mkdir build
+   $> cd build
+   $> cmake ..
+   $> make fpga_ip_export
+   ```
+
+   Windows:
+
+   ```bash
+   > cd add-oneapi
+   > mkdir build
+   > cd build
+   > cmake -G "NMake Makefiles" ..
+   > nmake fpga_ip_export
+   ```
+
+   For more details, see the Readme in the `add-oneapi` directory.
+
+2. **From the same terminal**, launch the Intel® Quartus® Prime Pro Edition GUI, and create a new Intel® Quartus® Prime project using the 'New Project' wizard. 
+
+   Linux:
+   
+   ```
+   $> cd ../../add-quartus
+   $> quartus
+   ```
+
+   Windows:
+   
+   ```
+   > cd ../../add-quartus
```

Comment:
Probably should use backslashes since you're in Windows

Reply:
good catch